toolchain/check/import_ref.cpp

```diff
@@ -977,6 +977,9 @@ static auto LoadLocalPatternConstantIds(ImportRefResolver& resolver,
 // LoadLocalPatternConstantIds(param_patterns_id) has completed without adding
 // any new work to work_stack_.
 //
+// `self_param_id` is an optional out parameter, populated with the InstId in
```

Comment:
The TODO below is about the paragraph above, I think? So maybe we should move this below the TODO paragraph?

Reply:
8a5d3e46040f023cfa0ed48b1848d26886fd680e